docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -89,6 +89,11 @@
           helm_chart_overrides: "/home/{{ ansible_ssh_user }}/{{ helm_chart_overrides | basename }}"
         when: helm_chart_overrides is defined
 
+      - set_fact:
+          helm_chart_overrides: "/usr/local/share/applications/overrides/wind-river-cloud-platform-deployment-manager-overrides-subcloud.yaml"
```

Comment:
Can we check the existence of these two files right after setting the facts? And fail the playbook with a meaningful error message?

Reply:
Tested the existence 
<img width="515" alt="image" src="https://github.com/Wind-River/cloud-platform-deployment-manager/assets/129255520/d5b8e054-929e-4adb-81d4-70b8c82fcf23">

Failing the playbook by deleting the overrides file.
<img width="743" alt="image" src="https://github.com/Wind-River/cloud-platform-deployment-manager/assets/129255520/beca9446-7a25-4324-adef-8f3dfd9a2f7d">